multiversx_sdk_core/transaction_builders/relayed_v1_builder.py

```diff
@@ -0,0 +1,132 @@
+import base64
+import json
+from typing import Any, Dict, Optional
+
+from multiversx_sdk_core.address import Address
+from multiversx_sdk_core.errors import ErrInvalidRelayerV1BuilderArguments
+from multiversx_sdk_core.interfaces import (IAddress, INetworkConfig, INonce,
+                                            ITransactionOptions,
+                                            ITransactionVersion)
+from multiversx_sdk_core.transaction import Transaction
+from multiversx_sdk_core.transaction_payload import TransactionPayload
+
+
+class RelayedTransactionV1Builder:
+    def __init__(
```

Comment:
JS implementation does not have a constructor with parameters:

https://github.com/multiversx/mx-sdk-js-core/blob/main/src/relayedTransactionV1Builder.ts

But all right to have it here, I think.

Reply:
Indeed. I was thinking that if you want to instantiate an object without calling all the methods to initialize it, you should be able to do that as well.